packages/components/src/index.js

```diff
@@ -29,7 +29,7 @@ export * from './Components/BulkSelect';
 export * from './Components/ConditionalFilter';
 export * from './Components/TagCount';
 export * from './Components/TagModal';
-export * from './Components/PrimaryToolbar';
+// export * from './Components/PrimaryToolbar';
```

Comment:
:question:

Reply:
Whoops, forgot to re-add it when testing some comments.